### PR TITLE
chore(kno-4899): clarify security requirements for in-app feed docs

### PR DESCRIPTION
**Pull request by brentjanderson**
### Description

Some customers have been confused about how to secure the notification feed. This adds relevant context to our in-app feed docs, linking back to the guide on securing the feeds.

KNO-4899

### CODE REVIEW

File: content/in-app-ui/ios/reference.mdx
Comment (JEverhart383) [inline content/in-app-ui/ios/reference.mdx:1]: I believe this needs to be HTML to render correctly, and should use the same language used in React Native and React references: https://github.com/knocklabs/docs/blob/bfc4dfc50f1a613cff4954bf22677585a9fac768/content/in-app-ui/react-native/reference.mdx#L38
Screenshot 2023-12-21 at 11 42 14 AM

File: content/in-app-ui/react-native/notification-feeds.mdx
Comment (JEverhart383) [inline content/in-app-ui/react-native/notification-feeds.mdx:1]: I like this callout, and the subsequent inclusion of the comment. This component doesn't look like it parses the MD and needs `<code></code>` tags and an `<a>` for the link to render correctly:
Screenshot 2023-12-21 at 11 44 31 AM

Comment (JEverhart383) [inline content/in-app-ui/react-native/notification-feeds.mdx:1]: Same as above

**Review by JEverhart383 — CHANGES_REQUESTED** [review]
I like calling out the security implications here, since that's an important step in making this production ready. Just a few comments about rendering and a suggestion on keeping language consistent across reference sections.

File: content/in-app-ui/ios/reference.mdx
Comment (brentjanderson) [reply content/in-app-ui/ios/reference.mdx:1]: Sure

> same language

Which language do you think we should use here? I was trying to keep it compact for this context, with the link as an invitation for the reader to go learn what enhanced security mode means.

Comment (JEverhart383) [reply content/in-app-ui/ios/reference.mdx:1]: Both the React and React Native reference files use this to describe the `userToken`: 
```
<span>
        A JWT that identifies the authenticated user, signed with the private
        key provided in the Knock dashboard. Required to secure your production
        environment.{" "}
        <a
          href="https://docs.knock.app/in-app-ui/security-and-authentication#authentication-with-enhanced-security-enabled"
          target="_blank"
        >
          Learn more.
        </a>
      </span>
```

Comment (brentjanderson) [reply content/in-app-ui/ios/reference.mdx:1]: Updated.

**Review by JEverhart383 — APPROVED** [review]
Had to mess with the spacing, but LGTM

**Comment by brentjanderson** [comment]
> Had to mess with the spacing, but LGTM

Thanks!

**Pull request merged**